c2rust-analyze/src/util.rs

```diff
@@ -348,3 +348,6 @@ pub fn is_null_const(constant: Constant) -> bool {
         _ => false,
     }
 }
+
+pub trait PhantomLifetime<'a> {}
```

Comment:
Should this be `trait PhantomLifetime<'a>: 'a {}` or something like that?  As-is, it seem like it doesn't constrain anything.

Reply:
I don't think so?  Someone suggested this online when I was searching up the lifetime error.  Adding a `: 'a` makes it not compile.  I think the point of the `PhantomLifetime` is just to mark that `'a` is used.  Just like there's

```rust
pub struct PhantomData<T: ?Sized>;
```